server_management/management/commands/_core.py

```diff
@@ -135,19 +136,62 @@ def get_remote(remote):
     return remote_prompt
 
 
-def title_print(title, state=''):
+def prompt(text, default=None, validate=None):
+    if default:
+        text = text.strip()
+        if text[-1] == ':' and len(text) > 1:
```

Comment:
As this function is only used internally, I'm not sure why you would strip a colon from the end if it is present - just don't use the function that way.